lib/protobuf/field/base_field.rb

```diff
@@ -127,11 +134,18 @@ def required?
       # FIXME: need to cleanup (rename) this warthog of a method.
       def set(message_instance, bytes)
         return message_instance[name] = decode(bytes) unless repeated?
+
+        if map?
```

Comment:
No. "Packed" can only be applied to repeated scalar numeric types. Map types are always repeated message fields on the wire.